app-emacs/posframe/posframe-1.0.4.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="27"
+
+inherit elisp
+
+DESCRIPTION="Pop a postframe at point."
+HOMEPAGE="https://github.com/tumashu/posframe"
+COMMIT="74f06b77deeb770cd0a96977b1e6bdedb682487a"
```

Comment:
_Personally_, I'd put this above HOMEPAGE and call it MY_COMMIT instead, but I don't think this is wrong as-is and some people have different opinions.

(Weak) rationale for each:
1. I like having the "real" standard variables together, then custom ones on top/below (not so much)
2. MY_COMMIT instead of COMMIT has an element of 'namespacing' in case something weird in the build system uses COMMIT or whatever.